utils/GlobalPOM.ts

```diff
@@ -0,0 +1,21 @@
+import { LoginPage } from "./components/loginPage";
+import { Page } from "@playwright/test";
+import { TopPanel } from "./components/topPanel";
+import { Dashboard } from "./components/dashboard";
+
+/*
+* Common actions for common Abstract Game functionality
+* */
+export class TestRailPage {
+    public loginPage: LoginPage;
```

Comment:
It's not a good pattern to use Global page classes like this. It's better to use separate page classes for each page and access them separatelly in the test

Reply:
I would like to be able to access all page elements in test scenarios, and how is this worse compared to using a specific class. It's just creating dependencies between classes and grouping them together.

Reply:
@voropa I would like to be able to access all page elements in test scenarios, and how is this worse compared to using a specific class. It's just creating dependencies between classes and grouping them together.